### ISSUE TITLE
HTTP/500 on /api/webhooks/grafana

### ISSUE THREAD

**Issue by n0guest** [guardian/alerta#392]
Hello.

Looks like Alerta 4.10.2 (+MongoDB 3.4 replicaSet) and Grafana 4.5.1 won't work together. Each alert-request from Grafana to Alerta ends with this:

```
HTTP/1.1 500 INTERNAL SERVER ERROR
Server: nginx
Date: Thu, 21 Sep 2017 00:18:41 GMT
Content-Type: application/json
Content-Length: 61
Connection: keep-alive
Access-Control-Allow-Origin: https://alerta.domain.tld
{
  "message": "$nin needs an array", 
  "status": "error"
}
```

And with help of https://requestb.in i got full copy of JSON-payload from Grafana:

```
{
  "evalMatches": [
    {
      "value": 97.007606,
      "metric": "user",
      "tags": {
        "__name__": "netdata_cpu_cpu_percentage_average",
        "chart": "cpu.cpu0",
        "dimension": "user",
        "family": "utilization",
        "instance": "zeta.domain",
        "job": "monitoring"
      }
    }
  ],
  "message": "boom!",
  "ruleId": 7,
  "ruleName": "Zeta CPU alert",
  "ruleUrl": "https://grafana.domain.tld/dashboard/db/alarms?fullscreen&edit&tab=alert&panelId=1&orgId=1",
  "state": "alerting",
  "title": "[Alerting] CPU alert"
}
```

Is it a bug? Or i'm missing something in Alerta deployment/configuration?

Thanks.

**Comment by satterly** [comment]
Looks like Grafana `tags` are actually a dictionary not a list as assumed. I've used your captured output as the new test data for this endpoint. Thanks.

**Comment by satterly** [comment]
Available now as version 4.10.3.

**Comment by n0guest** [comment]
> Available now as version 4.10.3. Please test.

Yep, it works now. Thank you.
